src/main/java/com/datatorrent/netlet/OptimizedEventLoop.java

```diff
@@ -75,9 +75,9 @@ public boolean add(SelectionKey key)
       }
       keys[pos++] = key;
       if (pos == keys.length) {
-        SelectionKey[] keys = new SelectionKey[this.keys.length << 1];
-        System.arraycopy(this.keys, 0, keys, 0, pos);
-        this.keys = keys;
+        SelectionKey[] lKeys = new SelectionKey[this.keys.length << 1];
```

Comment:
I only searched for local variables that start with `l` and I did not find any occurrences. I also searched for `this` and there are few places where instance fields are shadowed by method parameters.